redux/package.json

```diff
@@ -20,10 +20,11 @@
     ]
   },
   "dependencies": {
-    "keymirror": "^0.1.1",
-    "react": "^0.13.0",
```

Comment:
I had to remove `react` because otherwise it includes it twice in the bundle.
We can add it back after all other examples have local dependencies.